CHANGELOG.md

```diff
@@ -8,11 +8,14 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 - Added ConVar to toggle double-click buying
 - Added Japanese translation (by @Westoon)
+- Added Drag&Drop role layering VGUI, accessable with the console command `ttt2_edit_rolelayering`. Keep in mind that `ttt_[roleName]_random` and similar conditions are still taking place in the calculations. 
+  So if you wanna have a real "50/50" selection in a two-entry-sized "OR"-table layer, then you have to set the related `ttt_[roleName]_random` ConVars to 100%.
```

Comment:
I get what you are trying to do, but usage/explanations of changes shouldn't be documented in the changelog.
A page in our docs should be the way to go imo

```suggestion
- Added Drag&Drop role layering VGUI, accessible with the console command `ttt2_edit_rolelayering`
```

Reply:
Since this will be removed once integrated into the settings menu, I don't think it is that important

Reply:
Yea and the other two ToDos are something that has to be discussed but isn't needed currently / for this PR. Additionally, vgui improvements gonna come with/after the vgui PR.